packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -40,6 +40,7 @@ class ExpansionTile extends StatefulWidget {
     this.children = const <Widget>[],
     this.trailing,
     this.initiallyExpanded = false,
+    this.maintainsState = false,
```

Comment:
nit: in other places of the framework we call a similar setting "maintainState" without the "s". Can you rename this to be consistent?